### ISSUE TITLE
Issue while importing independent variable y from dataset

### ISSUE THREAD

**Issue by hemanshu-namdeo** [spyder-ide/spyder#8741]
### What steps will reproduce the problem?

import numpy as np
import matplotlib.pyplot as plt
import pandas as pd

dataset = pd.read_csv('Churn_Modelling.csv')

X = dataset.iloc[:,3:13].values
Y = dataset.iloc[:, 13].values

### Traceback
```python-traceback
  File "D:\Microsoft Azure\Microsoft Visual Studio\Shared\Anaconda3_64\envs\venv\lib\site-packages\spyder\widgets\variableexplorer\collectionseditor.py", line 482, in createEditor
    if not editor.setup_and_check(value, title=key, readonly=readonly):
  File "D:\Microsoft Azure\Microsoft Visual Studio\Shared\Anaconda3_64\envs\venv\lib\site-packages\spyder\widgets\variableexplorer\arrayeditor.py", line 620, in setup_and_check
    self.data.flags.writeable = True
ValueError: cannot set WRITEABLE flag to True of this array
```

## Versions

* Spyder version: 3.3.2 
* Python version: 3.6.8
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
pyflakes >=0.6.0  :  2.1.0 (OK)
pycodestyle >=2.3 :  2.5.0 (OK)
pygments >=2.0    :  2.3.1 (OK)
sphinx >=0.6.6    :  1.8.4 (OK)
rope >=0.9.4      :  0.11.0 (OK)
jedi >=0.9.0      :  0.13.2 (OK)
nbconvert >=4.0   :  5.3.1 (OK)
pandas >=0.13.1   :  0.24.1 (OK)
numpy >=1.7       :  1.16.1 (OK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.2.0 :  4.4.3 (OK)
IPython >=4.0     :  7.2.0 (OK)
matplotlib >=2.0.0:  3.0.2 (OK)
pylint >=0.25     :  2.2.2 (OK)
```

**Comment by hemanshu-namdeo** [comment]
When i double click on Y after importing dataset, i get an pop up 'Spyder has encountered an internal problem'

**Comment by ccordoba12** [comment]
Duplicate of issue #8582. It'll be fixed in our next version (3.3.3).
